https://sourceware.org/ml/libc-alpha/2017-08/msg00053.html

Add missing include for pthread_setcancelstate

* sysdeps/posix/shm_open.c: Include <pthread.h>.

diff --git a/sysdeps/posix/shm_open.c b/sysdeps/posix/shm_open.c
index 64315de8f8..eeb6cb5589 100644
--- a/sysdeps/posix/shm_open.c
+++ b/sysdeps/posix/shm_open.c
@@ -25,6 +25,7 @@
 #else
 
 # include <fcntl.h>
+# include <pthread.h>
 # include <shm-directory.h>
 
 
